homeassistant/components/ipp/strings.json

```diff
@@ -6,25 +6,25 @@
         "title": "Link your printer",
         "description": "Set up your printer via Internet Printing Protocol (IPP) to integrate with Home Assistant.",
         "data": {
-          "host": "Host or IP address",
-          "port": "Port",
+          "host": "[%key:common::config_flow::data::host%]",
+          "port": "[%key:common::config_flow::data::port%]",
           "base_path": "Relative path to the printer",
           "ssl": "Printer supports communication over SSL/TLS",
           "verify_ssl": "Printer uses a proper SSL certificate"
         }
       },
       "zeroconf_confirm": {
-        "description": "Do you want to add the printer named `{name}` to Home Assistant?",
+        "description": "[%key:common::config_flow::description::confirm_setup%]",
```

Comment:
After doing this, the corresponding `description_placeholders` in config flow could be removed.

Reply:
The common string still needs the name

Reply:
Am I looking at the wrong file/version? Seems that `confirm_setup` does not include `{name}`.
https://github.com/home-assistant/core/blob/71b797faf25f59d24f3469733883249ab24337a7/homeassistant/strings.json#L19-L25

Reply:
ah my bad i was looking at PR version again but another update was made that removed such name placeholder. Ill prob go back to the version i had for now rather than common string